packages/flutter/lib/src/widgets/primary_scroll_controller.dart

```diff
@@ -17,6 +17,8 @@ const Set<TargetPlatform> _kMobilePlatforms = <TargetPlatform>{
 
 /// Associates a [ScrollController] with a subtree.
 ///
+/// {@youtube 560 315 https://www.youtube.com/watch?v=33_0ABjFJUU}
```

Comment:
Can you add this one to ScrollView.primary as well? It's templated for every subclass with primary.